xbmc/guilib/GUIEditControl.cpp

```diff
@@ -747,3 +747,11 @@ void CGUIEditControl::SetFocus(bool focus)
   g_Windowing.EnableTextInput(focus);
   CGUIControl::SetFocus(focus);
 }
+
+std::string CGUIEditControl::GetDescriptionByIndex(int index) const
+{
+  if (index == 0)
+    return GetDescription();
+
+  return GetLabel2();
```

Comment:
In this instance it behaves as a a flag, but in the future, for controls with more than one alternate label (such as a fade label), it will make more sense.